Directory.Build.props

```diff
@@ -19,4 +19,9 @@
     </AdditionalFiles>
   </ItemGroup>
 
+  <!-- Get the ref assemblies from a package to avoid depending on the rf assemblies being available on the machine -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2" IncludeAssets="Build" PrivateAssets="All" />
```

Comment:
>Include [](start = 22, length = 7)

Are you going to use Packages.props?

Reply:
(Update=)

---
In reply to: [290073291](https://github.com/microsoft/MSBuildPrediction/pull/6#discussion_r290073291) [](ancestors = 290073291)

Reply:
Not for now. Central Package Versioning is more helpful for larger repos IMO. When there are only 2 projects, it doesn't help a ton :)